setup/ci/jenkins.md

```diff
@@ -121,29 +129,37 @@ In the UI, go to `"Manage Jenkins"` >> `"Configure System"` and set your git `us
 At this point, you should be able to manually run the script job in Jenkins
 (with parameters) and see it succeed.
 
-### Configuring Spinnaker
-Enable Igor.
+### Configure Spinnaker
 
-In spinnaker-local.yml, set:
+1. Make sure Igor is enabled.
 
-```
-jenkins.enabled = true
-jenkins.masters[0].name = <jenkins_name>
-jenkins.masters[0].address = http://<jenkins_host>/jenkins Note that "/jenkins" might not be the base path, it depends on how Jenkins is configured.
-jenkins.masters[0].username = <username>
-jenkins.masters[0].password = <user_api_token>
-```
+   Because you [enabled Jenkins](#add-your-jenkins-master), Igor should be
+   enabled automatically by Spinnaker. You can confirm this by checking your
+   `~/.hal/default/service-settings/`
```

Comment:
What should I see in here? Should it have created an `igor.yml`? (I still don't have one, though it appears Igor is enabled.)

Reply:
I think you would only create an igor.yml if you needed to explicitly override an igor enabled:false.

Sorry, I merged *just* before I saw this question.